sdk/search/test-resources.json

```diff
@@ -206,7 +213,7 @@
         },
         "SEARCH_KEYVAULT_URL": {
             "type": "string",
-            "condition": "[and(contains(variables('searchEncryptionLocations'), parameters('location')), not(equals(parameters('searchSku'), 'free')))]",
+            "condition": "[parameters('enableCmk')]",
```

Comment:
Just to make sure: `SEARCH_KEYVAULT_URL` is already treated as an optional recorded variable and code is conditioned suitably? I suspect since you're just changing the condition i.e., it was already conditional.

Reply:
Yes, "SEARCH_KEYVAULT_URL" is an optional recorded variable [here](https://github.com/Azure/azure-sdk-for-net/blob/03293ed1a353b3db03f646a9aa42de6821048384/sdk/search/Azure.Search.Documents/tests/Utilities/SearchTestEnvironment.cs#L86) and the code is also conditioned correctly [here](https://github.com/Azure/azure-sdk-for-net/blob/03293ed1a353b3db03f646a9aa42de6821048384/sdk/search/Azure.Search.Documents/tests/Samples/Sample06_EncryptedIndex.cs#L31-L34).